components/common/WalletModal.vue

```diff
@@ -24,32 +25,38 @@
         </b-field>
       </section>
       <section v-if="hasUserWalletAuth" class="modal-card-body">
-        <div class="has-text-centered">
-          <img
-            src="~/assets/Koda_Beta.svg"
-            alt="First NFT market explorer on Kusama and Polkadot"
-            height="32" />
-        </div>
-
-        <div v-show="!hasSelectedWalletProvider" class="buttons my-5">
+        <div v-show="!hasSelectedWalletProvider" class="buttons">
           <b-button
             v-for="(wallet, index) in wallets"
             :key="index"
             size="is-medium"
             icon-right="chevron-right"
             expanded
             @click="setWallet(wallet)">
-            <b-image
-              :src="wallet.img"
-              class="is-24x24"
-              style="display: inline-block; vertical-align: middle"></b-image>
-            {{ wallet.name }}
+            <span class="pl-4">
+              <b-image
+                :src="wallet.img"
+                class="is-32x32"
+                style="display: inline-block; vertical-align: middle"></b-image>
+              <span class="is-size-6">{{ wallet.name }}</span>
+            </span>
           </b-button>
         </div>
 
         <div
-          v-show="hasSelectedWalletProvider && !hasWalletProviderExtension"
-          class="buttons my-5">
+          v-if="hasSelectedWalletProvider && !hasWalletProviderExtension"
+          class="buttons">
+          <div
+            class="subtitle is-size-6 has-text-centered is-lowercase mb-0 pt-4 pb-4"
+            style="width: 100%">
+            <b-image
+              :src="selectedWalletProvider.img"
+              class="is-32x32"
+              style="display: inline-block; vertical-align: middle" />
```

Comment:
use bulma class instead of inline-block style. maybe you even find one for vertical-align too